README.md

```diff
@@ -27,7 +27,7 @@ On these systems installation is easy:
 2. Download and install `NetworkLocation.apk` as a usual app (you may need to enable "Unknown sources" in Settings->Security)
 3. Reboot and continue at [Usage](#usage)
 
-**Important:** Installing as system app will always works, instead installing as user app will only works out of the box on Android versions from 4.4 to 6.x.  
+**Important:** Installing as system app will always works, instead installing as user app will only work out of the box on Android versions from 4.4 to 6.x.  
```

Comment:
Slightly improve grammar:

> Installing as a system app will always work for all versions of Android. But installing as user app will only work on Android versions 4.4 to 6.x.

Reply:
I have kept "out of the box" because it mean without changes, you can make it working (but not out of the box).

Reply:
Also strike the "s" in "will always works" (→ "will always work") :)

Reply:
I have already corrected it when I have talked about "out of the box".